functions/taskrouterListeners/transfersListener.private.ts

```diff
@@ -29,6 +31,11 @@ type TransferMeta = {
   transferStatus: 'transferring' | 'accepted' | 'rejected';
 };
 
+type ChatTransferTaskAttributes = {
+  transferMeta?: TransferMeta;
+  transferTargetType: 'worker' | 'queue';
+};
+
 const isChatTransfer = (
```

Comment:
Should `taskAttributes.transferMeta.transferStatus === 'accepted'` be removed from this function? Or renamed the function?